### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by CormanYT** [SebOuellette/Discord-LiveBot#37]
This is a **lot** of suggestions. I don't expect you to get them all done quickly, but here are some ideas. I'd focus on the important ones and the ones the most people want first though.

1. Looking at Messages
- Add a way to show embed messages (like ones from bots)
- Add a way to show webhook messages
- Add a way to show images
2. Sending Messages
- Add a way to send images
3. Ways of Quick Information
- Add the ability to offer over a server to see it's name
- Add the ability to left click on a user (their message, or their name) and see their logo, username, roles, and an easy way to DM them
- Add a way to right click on a user to have quick options like call or DM, or copy ID 
4. Direct Messages
- Add in DMs
5. Voice Channels
- Add in a way to enter voice channels
6. Categories
- Add in a way to open or close categories
7. Administration
- Add in the Server Settings option
- Add in a convenient way to make channels and categories
8. Emojis and Reactions
- Add in a way to see emojis
- Add in a way to easily send emojis
- Add in a way to react to messages
- Add in a way to view the people reacting to a message and the reaction counts of each reaction on a message

**Comment by dcrime** [comment]
- Embedded messages are already a thing tho not the best looking
- Dm support #20
- Bots can't call add anyone/ call anyone / join any group calls, they're limited to servers strictly
- Voice channels #29
- Emojis #31

If you didn't read the other issues then please read them as for example dms have been suggested multiple times by now

**Comment by dcrime** [comment]
If you wish you can download this for now until the creator starts working on the project
https://github.com/dcrime/Discord-LiveBot it only has the master branch, didn't look at how to make the executable

**Comment by CormanYT** [comment]
Actually, Voice channels are possible. Not voice calls, but certainly voice channels, since I've seen bots play music on a user command, and bots should also be able to pick up voice channel audio.

**Comment by CormanYT** [comment]
Either way, this is just a list of features I think at **some point** (I code too, so I know adding these features is time-consuming) you should add.

**Comment by dcrime** [comment]
Yeah, that's possible and I've actually tried to do it in the past but that's another story haha, if you want we can try and do it together since yeah, there haven't been any updates in some time, my discord is daniel_crime#1551

**Comment by SebOuellette** [comment]
Embeds were pretty much finalized in commit 55fd9a4807eba08a7badb61696397ae324f68818, but I'd recommend downloading the latest Livebot version of course. 
Collapsable categores added in 407c348f40d58dd60c3d5c485d2282090bdfaabc

**Comment by dcrime** [comment]
Dm support has been added in commit 6e3fa34 though you can only access it through the right click menu or by getting the user through the console and opening the chat with them, for now that's the only way to do it Snippet: `dmChannelSelect(bot.users.cache.find(id))`

**Comment by dcrime** [comment]
I wanted to go over a few more
- Hover over server to display name has been added
- There's a right click menu that allows you to dm others, copy their id and for messages copy the message link
- Dms have been added though not the best looking or optimized that well
- Emojis can be seen, normal or custom
